plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -81,7 +81,12 @@ export function getQuerySpanName(
 }
 
 function parseNormalizedOperationName(queryText: string) {
```

Comment:
your commit message says you're changing how to get the "span name", but your changes are on the "operation name" function (even if that is later used to get the span), so probably worth updating commit message/PR title to reflect that.

Reply:
Done